config/application.rb

```diff
@@ -31,5 +31,13 @@ class Application < Rails::Application
     # custom
 
     config.autoload_paths << Rails.root.join("lib")
+
+    # Faraday, by default, writes logs at the :info log level, and logs request headers.
+    #
+    # Unfortunately this logs Authorization headers, so we disable the request header logging.
+    config.x.registry_request_log_options = {
```

Comment:
Let's put that into `config/initializers/settings.rb` please.

I'd even make it two options to be configurable on deployment level:

```ruby
config.x.registry_log_level   = Config.get(name: "REGISTRY_LOG_LEVEL", default: "info").to_sym
config.x.registry_log_headers = Config.get(name: "REGISTRY_LOG_HEADERS", default: "false").in? %w(1 true yes)
```

Those new options need to be added to the [documentation](https://github.com/klausmeyer/docker-registry-browser/blob/master/docs/README.md#configuration) then of course.

Reply:
Done, though I kept the `config` value as a single variable to allow for easy future configuration of additional `options` of the Faraday logger. It also makes the implementation more concise.

Reply:
That's fine :)